### PR TITLE
Removing "Shade"

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @bradtraversy 

removing "Shade" as project doesn't seem to exist anymore.

Cheers!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -427,7 +427,6 @@
 | [W3Layouts](https://w3layouts.com/)| W3Layouts: 3784+ Free Website Templates for 2020 |
 | [Tooplate](https://www.tooplate.com/)| Tooplate: Free HTML Templates for everyone! |
 | [Cruip](https://cruip.com/free-templates/)| Fully coded HTML templates to help you easily build your startup landing page without hassles. |
-| [Shade](https://grayic.com/shade/) | Get High Converting Landing Pages For Free! |
```
Comment (Alaev) [inline readme.md:430]: Looks like the website is under construction, let's give it another month perhaps?

Comment (spekulatius) [reply readme.md:430]: Sure, no problem. We can just leave the PR if you want. If it's still not online by the end of the year maybe it's time to merge it.

Comment (Alaev) [reply readme.md:430]: 🚀

Comment (JDsWebService) [reply readme.md:430]: Website is still down. I vote for removal.

**Comment by Alaev** [comment]
Removing from list, 22 days past.

**Review by Alaev — APPROVED** [review]

**Comment by spekulatius** [comment]
Thanks for merging @Alaev

**Pull request merged**